core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -1070,9 +1070,9 @@ public void terminateWorkflow(Workflow workflow, String reason, String failureWo
 				});
 				List<String> urns = (List<String>) map.get("urns");
 				for (String urn : urns) {
-					if (urn.contains("orderid:")) {
+					if (urn.contains("orderid:") || urn.contains("order:")) {
```

Comment:
We have a handy way of checking this .. lookup StatusEventPublisher.getJobId
May be we move instances of all those to a Util class and use it